### PR TITLE
52 create redux saga sample

### PR DESCRIPTION
**Pull request by brauliodiez**
Starting from sample 10 Async_Calls_Redux_Thunk, we have replaced Thunk by Sagas, important thing to notice: tsconfig output changed to ES6.

Pending in a future sample add unit test support to this (that will be a included in a separate sample).

@pablonete , @Nasdan , @JaimeSalas if you could take a look to it that would be great.

### CODE REVIEW

File: 15 AsyncCalls_Redux_Saga/package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    "lodash": "^4.5.1",
+    "object-assign": "^4.0.1",
+    "q": "^1.4.1",
+    "react": "~0.14.7",
+    "react-dom": "^0.14.7",
+    "react-redux": "^4.4.0",
+    "react-router": "^2.0.0",
+    "redux": "^3.3.1",
+    "redux-saga": "^0.10.4",
+    "redux-thunk": "^2.0.1",
```
Comment (Nasdan) [inline 15 AsyncCalls_Redux_Saga/package.json]: Is redux-thunk to uninstall?

File: 15 AsyncCalls_Redux_Saga/src/components/about/aboutPage.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+export default class About extends React.Component<Props, {}> {
+   public render() {
+     return (
+         <div className="row about-page top-buffer">
+           <h1 className="jumbotron">15 Redux Saga</h1>
+
+           <div className="col-xs-12">
+             <h1>
+               <small>
+                 This sample takes as starting point sample "10 AsyncCalls_Redux_Thunk".
```
Comment (Nasdan) [inline 15 AsyncCalls_Redux_Saga/src/components/about/aboutPage.tsx]: sample "15 AsyncCalls_Redux_Saga"

File: 15 AsyncCalls_Redux_Saga/package.json (hunk 1 above)
Comment (brauliodiez) [inline 15 AsyncCalls_Redux_Saga/package.json]: Yups, right

File: 15 AsyncCalls_Redux_Saga/src/components/about/aboutPage.tsx (hunk 2 above)
Comment (brauliodiez) [inline 15 AsyncCalls_Redux_Saga/src/components/about/aboutPage.tsx]: Starting point...

Comment (Nasdan) [inline 15 AsyncCalls_Redux_Saga/src/components/about/aboutPage.tsx]: Ah sorry

File: 15 AsyncCalls_Redux_Saga/src/components/app.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { createStore, applyMiddleware } from 'redux';
+import { Provider } from 'react-redux';
+import reduxThunk from 'redux-thunk';
```
Comment (Nasdan) [inline 15 AsyncCalls_Redux_Saga/src/components/app.tsx]: remove import

File: 15 AsyncCalls_Redux_Saga/src/components/app.tsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+import Header from './common/header';
+import reducers  from '../reducers';
+import createSagaMiddleware from 'redux-saga'
```
Comment (Nasdan) [inline 15 AsyncCalls_Redux_Saga/src/components/app.tsx]: semicolon

File: 15 AsyncCalls_Redux_Saga/src/sagas/membersSaga.ts [hunk 5]
```diff
@@ -0,0 +1,17 @@
+import { takeEvery, takeLatest } from 'redux-saga'
```
Comment (Nasdan) [inline 15 AsyncCalls_Redux_Saga/src/sagas/membersSaga.ts]: not used import

File: 15 AsyncCalls_Redux_Saga/src/components/app.tsx (hunk 3 above)
Comment (brauliodiez) [inline 15 AsyncCalls_Redux_Saga/src/components/app.tsx]: done

File: 15 AsyncCalls_Redux_Saga/src/components/app.tsx (hunk 4 above)
Comment (brauliodiez) [inline 15 AsyncCalls_Redux_Saga/src/components/app.tsx]: done

File: 15 AsyncCalls_Redux_Saga/src/sagas/watchersSaga.ts [hunk 6]
```diff
@@ -0,0 +1,19 @@
+import { takeLatest } from 'redux-saga'
+import { call, put, fork } from 'redux-saga/effects'
```
Comment (Nasdan) [inline 15 AsyncCalls_Redux_Saga/src/sagas/watchersSaga.ts]: not used call and put

File: 15 AsyncCalls_Redux_Saga/src/sagas/watchersSaga.ts [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+import {fetchMembers} from './membersSaga';
+
+// add here all your watchers
+// https://github.com/yelouafi/redux-saga/blob/master/examples/real-world/sagas/index.js
+function* watchLoadMembersRequest() {
+  while(true) {
```
Comment (Nasdan) [inline 15 AsyncCalls_Redux_Saga/src/sagas/watchersSaga.ts]: I think this while it's not necessary

**Pull request merged**